impl/client/src/main/doc-js/pentaho/type/action/ITarget.jsdoc

```diff
@@ -17,7 +17,10 @@
 /**
  * The `ITarget` interface represents the entity _where_ an
  * [action]{@link pentaho.type.action.Base} takes place,
- * or the pacient/object of action.
+ * or the patient/object of action.
```

Comment:
The _agent_ acts on the _patient_. It's the most correct, direct way to say — not quite metaphor. "object", on the other hand is more tied to the grammatical subject and object classification. It happens that depending on the verb, the agent and patient may be the subject and the object or the other way round.

All that said, I do realize that the "patient" word, in this context, is not very well known.
I would preferably remove "object".